projects/laji/src/app/shared-modules/search-filters/three-state-multi-switch/switch-row/switch-row.component.ts

```diff
@@ -17,6 +17,7 @@ export class SwitchRowComponent implements OnChanges {
   @Output() update = new EventEmitter<{id: string, value: any}>();
 
   state;
+  typeCheckbox: number;
```

Comment:
instead of number this should be typed to use the CheckboxType Enum (using the enum is more clear)